packages/data-service-generator/src/admin/create-dotenv.ts

```diff
@@ -85,3 +90,35 @@ function removeDuplicateKeys(arr: VariableDictionary): VariableDictionary {
   });
   return Array.from(variablesMap, ([key, value]) => ({ [key]: value }));
 }
+
+function sortAlphabetically(arr: VariableDictionary): VariableDictionary {
```

Comment:
@Meyazhagan please add any relevant unit test for any new logic added

Reply:
done